src/url_json.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+#
+# igcollect - JSON data from URL
+#
+# Copyright (c) 2017, InnoGames GmbH
+#
+
+from argparse import ArgumentParser
+from time import time
+try:
+    from urllib.request import urlopen
+except ImportError:
+    from urllib2 import urlopen
+import json
+
+
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument('--prefix', default='data')
+    parser.add_argument('--url', default='http://localhost/')
+    parser.add_argument('--keys', default='key1,key2')
```

Comment:
It would better be a multi argument with `action='append'`.  You would then get the keys in a `list`.